_layouts/homepage.html

```diff
@@ -21,18 +21,13 @@ <h1 class="Page-title">The South African National Budget {{ page.financial_year
       <em>vulekamali</em> is a project by the South African National Treasury and Imali Yethu with the goal of making government budget data and processes accessible to all citizens & interested parties. <a href="/about">Read more about vulekamali.</a>
     </div>
 
-    <div class="Alert u-marginBottom20">
-      <div class="Alert-content">
-        {%
-          include components/universal/Icon/index.html
-          type="alert"
-          size="large"
-        %}
-        <span class="Alert-heading">
-          Please note
-        </span>
-      </div>
-      <div class="Alert-text">The full Medium Term Budget Policy Statement is <a href="http://www.treasury.gov.za/documents/mtbps/2018/default.aspx">available on National Treasury's website</a>. The MTBPS documents and data will be added to vulekamali.gov.za soon.</div>
+    <div class="u-marginBottom20">
+      {%
```

Comment:
This goes against what we did in https://github.com/OpenUpSA/static-budget-portal/pull/405/files#diff-6d5885cbf9d64c48d9766289aff8666eR24 where you just included the raw HTML. I still think you're making life hard for yourself turning this into a component -- it's an unnecessary abstraction that doesn't buy you much, and essentially is a thin wrapper over really simple HTML. Your component now has a number of conditionals, which could be eliminated.

Reply:
I think the second code set is easier to read -- I don't need to unwrap a layer of abstraction, and I can see that I can put plain old HTML wherever I like, including links (which was the issue last time). The more abstractions we put over "plain old HTML", the harder we have to think about extensibility, re-use, etc. It's okay to repeat HTML.